calibration:
  bottom_joints:
    - - 0.39939437674748185
      - 0.04007607543772887
      - 0.03393
    - - -0.16499028896068607
      - 0.36592371411083663
      - 0.03393
    - - -0.23440408778679558
      - 0.3258476386731078
      - 0.03393
    - - -0.23440408778679558
      - -0.3258476386731078
      - 0.03393
    - - -0.16499028896068607
      - -0.36592371411083663
      - 0.03393
    - - 0.39939437674748185
      - -0.04007607543772887
      - 0.03393
  bottom_transform:
    - 0.0
    - 0.0
    - 0.2
    - 0.0
    - 0.0
    - 0.0
    - 1.0
  fixed_leg_length:
    - 0.6301635287194699
    - 0.6301616783528776
    - 0.6301613983302564
    - 0.6301610031514615
    - 0.630159836926982
    - 0.6301618478684873
  platform_joints:
    - - 0.17819884557545404
      - 0.26488815363389784
      - -0.035
    - - 0.14030044742078387
      - 0.2867688040103523
      - -0.035
    - - -0.3184992929962378
      - 0.021880650376454557
      - -0.035
    - - -0.3184992929962378
      - -0.021880650376454557
      - -0.035
    - - 0.14030044742078387
      - -0.2867688040103523
      - -0.035
    - - 0.17819884557545404
      - -0.26488815363389784
      - -0.035
  platform_transform:
    - 0.0
    - 0.0
    - 0.8144899999999999
    - 0.0
    - 0.0
    - 0.0
    - 1.0
